RELEASE.md

```diff
@@ -212,6 +212,42 @@ Rename `Dangerzone.msi` to `Dangerzone-$VERSION.msi`.
 
 Linux binaries are automatically built and deployed to repositories when a new tag is pushed.
 
+### Fedora
+
+Because the Fedora packages do not contain compiled Python code for a specific
+Python version, we can create a single Debian package and use it for all of our
+Debian-based distros.
```

Comment:
This section needs to be reworded.

Reply:
Whoops. This was some copy-pasta mistake.

Reply:
I guess the note that I added bellow was supposed to replace this, but I forgot to remove it in the end, I guess. Removing it now.